apps/package.json

```diff
@@ -59,7 +59,7 @@
     "@code-dot-org/maze": "2.12.0",
     "@code-dot-org/ml-activities": "0.0.26",
     "@code-dot-org/ml-playground": "0.0.37",
-    "@code-dot-org/p5.play": "^1.3.18-cdo",
+    "@code-dot-org/p5.play": "^1.3.17-cdo",
```

Comment:
i know you confirmed that this version fixes the issue in code-dot-org, but will rolling back to this version reintroduce a bug in the alpha/tinting logic? (this is a sanity check, not a trick question 😄 )

Reply:
Yes, we will lose all the polish things that came from moving forward in the alpha tint. So this means, there won't be a default value for alpha, tint and alpha won't be able to be applied together, and alphas at 0 will result in a fully visible, rather than invisible sprite. Overall, this seems less bad than fully broken projects, but it will mean we'll need to go back and re-apply these changes. My current plan is to re-apply the second and third of those three changes, since my current understanding is that the root of this issue is in the first.

Reply:
agreed -- i couldn't remember what changes were in the new version, but i think that trade-off makes sense here